other/plague/plague carrier

```diff
@@ -1,21 +1,26 @@
 **Plague Carrier** | Solo Miscellaneous - Plague Team
 __Basics__ 
-The Plague Carrier wins with the plague bearers and joins a channel with them to communicate securely.
+The Plague Carrier learns which roles will die to the plague.
```

Comment:
I prefer the phrasing towards the end of the desc "At the start of every night, the Plague Carrier is told the roles of players who will die from the plague that night." quite a bit, as roles technically don't die. Maybe we can change it to be more like that here?

Reply:
That's a good point